src/Diagnostics.DataProviders/Exceptions/KustoResponseSchemaException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Diagnostics.DataProviders.Exceptions
+{
+    public class KustoResponseSchemaException : Exception
+    {
+        public KustoResponseSchemaException(string sourceName, string message): base($"Exception occurred in {sourceName}. {message}")
```

Comment:
VS should let you generate all inherited constructors from the base class - I believe there are 3 standard public exception constructors, they should be available on custom exceptions